client/src/pages/Signup.jsx

```diff
@@ -62,7 +62,7 @@ const Signup = () => {
   return loading ? (
     <p>Loading...</p>
   ) : (
-    <div className="flex flex-col items-center justify-center min-h-screen bg-gray-100">
+    <div className="flex flex-col items-center justify-center min-h-screen bg-gray-100 text-black">
```

Comment:
The text background is also black.
```suggestion
    <div className="flex flex-col items-center justify-center min-h-screen bg-gray-100">
```

https://github.com/user-attachments/assets/695a5dde-9a8c-4702-8176-b45c1441254b